web/ts/TUMLiveVjs.ts

```diff
@@ -157,6 +157,7 @@ export const initPlayer = function (
     courseUrl?: string,
     streamStartIn?: number, // in seconds
 ) {
+    const nativeTextTracks = (/iPad|iPhone|iPod/.test(navigator.userAgent));
```

Comment:
This seems like what ESLint wants
```suggestion
    const nativeTextTracks = /iPad|iPhone|iPod/.test(navigator.userAgent);
```